From b57b7d3e586f3e12b7bdbc552d4fd3e904174168 Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Mon, 23 Oct 2017 09:21:46 +0800
Subject: [PATCH] arm64: dts: rockchip: move PCIe configuarion into
 rk3399-excavator-sapphire.dtsi

So all the variant boards will benefit from it and no need
to copy and paste this everywhere.

Change-Id: Id27355bb6f780ead76b8661693c32a452d3f8e61
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 .../arm64/boot/dts/rockchip/rk3399-excavator-sapphire.dtsi | 7 +++++++
 .../boot/dts/rockchip/rk3399-sapphire-excavator-edp.dts    | 4 ----
 2 files changed, 7 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-excavator-sapphire.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-excavator-sapphire.dtsi
index c7446fc00e9f..344b2a53c777 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-excavator-sapphire.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-excavator-sapphire.dtsi
@@ -172,6 +172,13 @@
 	};
 };
 
+&pcie0 {
+	ep-gpios = <&gpio2 4 GPIO_ACTIVE_HIGH>;
+	num-lanes = <4>;
+	pinctrl-names = "default";
+	pinctrl-0 = <&pcie_clkreqn>;
+};
+
 &uart0 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&uart0_xfer &uart0_cts>;
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dts b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dts
index 582bb40cd26b..ae154890b108 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dts
@@ -276,10 +276,6 @@
 };
 
 &pcie0 {
-	ep-gpios = <&gpio2 4 GPIO_ACTIVE_HIGH>;
-	num-lanes = <4>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pcie_clkreqn>;
 	status = "okay";
 };
 
-- 
2.35.3

